Books/render-line-item.js

```diff
@@ -0,0 +1,31 @@
+import { calcLineTotal } from './common/utils.js';
+import { toUSD } from './common/utils.js';
+
+function renderLineItem(lineItem, book) {
+    const tr = document.createElement('tr');
+
+    const nameCell = document.createElement('td');
+    nameCell.className = 'align-left';
```

Comment:
I don't see an `align-left` class in your `shopping-cart.css` file. Make sure to check you're adding the right class.